server/src/main/java/io/spine/server/commandbus/CommandAckMonitor.java

```
@@ -71,20 +65,27 @@ private CommandAckMonitor(Builder builder) {
     /**
      * {@inheritDoc}
      *
-     * <p>Posts either {@link CommandAcknowledged} or {@link CommandErrored} system
-     * event depending on the value of the given {@code Ack}.
-     *
-     * @param value
+     * <p>Posts either {@link CommandAcknowledged}, {@link CommandErrored}, or
+     * {@link CommandRejected} system event depending on the value of the given {@code Ack}.
      */
     @Override
     public void onNext(Ack value) {
-        super.onNext(value);
         postSystemEvent(value);
     }
 
+    @Override
+    public void onError(Throwable t) {
+        // NO-OP.
```

Same as above. Shouldn't we rethrow the exception?